### PR TITLE
[frappe/gantt] added language option

### PR DESCRIPTION
**Pull request by TempleClause**
Added the language option:
https://github.com/frappe/gantt/blob/8f0b83d27d6e5f970dd4e5125120e70a3374b923/src/index.js#L89

### CODE REVIEW

**Review by samalexander — APPROVED** [review]

**Review by samalexander — CHANGES_REQUESTED** [review]
@TempleClause please can you add the option to the tests file.

**Comment by TempleClause** [comment]
@samalexander 

like so?

File: types/frappe-gantt/frappe-gantt-tests.ts [hunk 1]
```diff
@@ -36,4 +36,5 @@ new Gantt('#gantt', tasks, {
         </div>
       `;
     },
+    language: "es",
```
Comment (samalexander) [inline types/frappe-gantt/frappe-gantt-tests.ts:39]: That's fine, but please change to single quotes (to match the rest of the tests code).

**Review by samalexander — CHANGES_REQUESTED** [review]

File: types/frappe-gantt/frappe-gantt-tests.ts (hunk 1 above)
Comment (TempleClause) [reply types/frappe-gantt/frappe-gantt-tests.ts:39]: done

**Review by samalexander — APPROVED** [review]

**Comment by TempleClause** [comment]
Ready to merge

**Pull request merged**